index.js

```diff
@@ -6,11 +6,19 @@ const { GitHub, context } = require('@actions/github')
 const main = async () => {
   const token = core.getInput('github-token')
   const number = core.getInput('number')
+  const repo = core.getInput('repo')
+
+  var repoValue
+  if (!repo) {
+    repoValue = context.repo
+  } else {
+    repoValue = repo.split('/')
+  }
```

Comment:
```suggestion
  const repo = core.getInput('repo')?.split('/') || context.repo
```

This could be simplified a bit using the optional chaining parameter.

However, I think the `repo.split('/')` will create the wrong results, as it returns an array `['owner', 'repo']` while we need an object `{ owner, repo }` to pass to `createReview()`.

Reply:
This is exacts what the context does:
https://github.com/actions/toolkit/blob/e2eeb0a784f4067a75f0c6cd2cc9703f3cbc7744/packages/github/src/context.ts#L66

Reply:
Oh, nm, next line

Reply:
```suggestion
  const repo = core.getInput('repo')

  var repoValue
  if (!repo) {
    repoValue = context.repo
  } else {
    const [owner, repo] = repo.split('/')
    repoValue = {owner, repo}
  }
```